libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/AztecEditorFragment.java

```diff
@@ -246,6 +265,18 @@ public boolean onOptionsItemSelected(MenuItem item) {
         return false;
     }
 
+    @Override
+    public void onRedoEnabled() {
```

Comment:
The `onRedoEnabled()` and `onUndoEnabled()` methods are triggered when the history changes (i.e. [History.handleHistory](https://github.com/wordpress-mobile/WordPress-Aztec-Android/blob/develop/aztec/src/main/kotlin/org/wordpress/aztec/History.kt#L24), [History.redo](https://github.com/wordpress-mobile/WordPress-Aztec-Android/blob/develop/aztec/src/main/kotlin/org/wordpress/aztec/History.kt#L55), and [History.undo](https://github.com/wordpress-mobile/WordPress-Aztec-Android/blob/develop/aztec/src/main/kotlin/org/wordpress/aztec/History.kt#L88)).  Changing that would have to take place in the library (i.e. Aztec) rather than the client (i.e. WordPress-Android).  I created an [issue](https://github.com/wordpress-mobile/WordPress-Aztec-Android/issues/294) for in the library repository.